mne/viz/_brain/_brain.py

```diff
@@ -2537,7 +2537,7 @@ def remove_skull(self):
     @fill_doc
     def add_volume_labels(
         self,
-        aseg="aparc+aseg",
+        aseg="wmparc",
```

Comment:
This will be a backward incompatible change with no deprecation cycle. To make it backward compatible, we could have `aseg="auto"` which means "'aparc+aseg' if it's there and otherwise 'wmparc', and if neither is there, raise an error"

We should also make similar changes to `get_montage_volume_labels` for example.